bin/eslint-github-init.js

```diff
@@ -47,7 +47,7 @@ const questions = [
   }
 ]
 
-inquirer.prompt(questions).then(answers => {
+for (const answers of inquirer.prompt(questions)) {
```

Comment:
does this need an `await`?

Reply:
Oh shoot, this was supposed to be a `await/async` conversion and not a `for-of` one.